demos/human_pose_estimation_demo/README.md

```diff
@@ -29,16 +29,24 @@ InferenceEngine:
 human_pose_estimation_demo [OPTION]
 Options:
 
-    -h                         Print a usage message.
-    -i                         Required. An input to process. The input must be a single image, a folder of images or anything that cv::VideoCapture can process.
-    -loop                      Optional. Enable reading the input in a loop.
-    -m "<path>"                Required. Path to the Human Pose Estimation model (.xml) file.
-    -d "<device>"              Optional. Specify the target device for Human Pose Estimation (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The application looks for a suitable plugin for the specified device.
-    -pc                        Optional. Enable per-layer performance report.
-    -no_show                   Optional. Do not show processed video.
-    -black                     Optional. Show black background.
-    -r                         Optional. Output inference results as raw values.
-    -u                         Optional. List of monitors to show initially.
+    -h                        Print a usage message.
+    -at "<type>"              Required. Type of the network, either "ae" for
+                              Associative Embedding or "openpose" for OpenPose.
```

Comment:
why not just 'op'?

Reply:
The "openpose" is not so long as "associative embedding", therefore I think the word reduction is not necessary here.